### PR TITLE
(doc) Add CCM 0.11.0 release notes

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description Of Changes

Release notes for upcoming version of Chocolatey Central Management 0.11.0.

## Motivation and Context

We need release notes.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before 
submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/central-management/release-notes.md [hunk 1]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
 
+## 0.11.0 (TBD)
+
+### Features
+
+- Add the ability to Import and Export Deployment Plan Definitions.
+- Add the ability to export the installed software from a computer to packages.config file - see [Licensed #355](https://github.com/chocolatey/chocolatey-licensed-issues/issues/355).
+- Add retention policies for completed and archived Deployment Plans.
+- Add computers reporting into Chocolatey Central Management to an 'All Computers' group.
+- Add the ability to send email notifications when a deployment plan has finished - see [Licensed #328](https://github.com/chocolatey/chocolatey-licensed-issues/issues/328).
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:42]:
```suggestion
- Add the ability to send email notifications when a Deployment Plan has finished - see [Licensed #328](https://github.com/chocolatey/chocolatey-licensed-issues/issues/328).
```

File: input/en-us/central-management/release-notes.md [hunk 2]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+
+### Bug Fixes
+
+- Fix - On the Deployment Step Details screen, when showing the contents of the log, the error highlighting doesn't extend to the total available width.
+- Fix - When viewing the log for a Deployment Step, the text disappears from the Copy button when the mouse is hovered over it.
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:47]:
```suggestion
- Fix - When viewing the log for a Deployment Step, the text disappears from the `Copy` button when the mouse is hovered over it.
```

File: input/en-us/central-management/release-notes.md [hunk 3]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Fix - Errors are reported in the Deployment Step Details log when no errors have occurred.
+- Fix - Audit log retention does not clean up old log entries - see [Licensed #336](https://github.com/chocolatey/chocolatey-licensed-issues/issues/336).
+
+### Improvements
+
+- Add an option to provide a default value for the `executionTimoutInSeconds` setting, which will be used for all new Deployment Steps.
+- Ensure a computer's `ComputerName` and `FriendlyName` properties are in separate fields when the API returns data.
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:54]:
```suggestion
- Ensure a Computer's `ComputerName` and `FriendlyName` properties are in separate fields when the API returns data.
```

File: input/en-us/central-management/release-notes.md [hunk 4]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Move the `Action` button to the first column in all tables.
+- Reinstate the ‘Return To Deployments’ button when editing a deployment plan.
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:56]:
```suggestion
- Reinstate the ‘Return To Deployments’ button when editing a Deployment Plan.
```

File: input/en-us/central-management/release-notes.md [hunk 5]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Remove the ability to assign any user the `ChocoAdmin` role.
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:57]:
```suggestion
- Remove the ability to assign a User the `ChocoAdmin` Role.
```

**Review by gep13 — CHANGES_REQUESTED** [review]
A few minor tweaks, otherwise, this LGTM!

File: input/en-us/central-management/release-notes.md [hunk 6]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
 - Please see our [GitHub repository issues](https://github.com/chocolatey/chocolatey-licensed-issues/labels/CentralManagement).
 - Some issues may be held internally, please follow your support routes to learn more.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:32]: Why do we have this here? All resolved  issues should be on the release notes so this shouldn't be necessary.

**Review by pauby — COMMENTED** [review]
This is offset of the current release, but I'm curious about why this is needed. Let's not put more pressure on the Support team.

File: input/en-us/central-management/release-notes.md (hunk 6 above)
Comment (gep13) [reply input/en-us/central-management/release-notes.md:32]: This is a hold over from the initial import of the release notes from the old location.  It has been around for quite a while, including in some of the other Product release notes.  Happy to remove this, but we should remove from all the places that it is located.

File: input/en-us/central-management/release-notes.md [hunk 7]
```diff
@@ -31,6 +31,32 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
 
+## 0.11.0 (TBD)
+
+### Features
+
+- Add the ability to Import and Export Deployment Plan Definitions.
```
Comment (gep13) [inline input/en-us/central-management/release-notes.md:38]:
```suggestion
- Add the ability to import and export Deployment Plan definitions.
```

Comment (gep13) [reply input/en-us/central-management/release-notes.md:38]: Just so that we are consistent with the line below

**Review by gep13 — CHANGES_REQUESTED** [review]
Just one final change.

**Comment by gep13** [comment]
Also, we will need to update the `TBD` with the actual release date.

**Review by gep13 — APPROVED** [review]
LGTM!

File: input/en-us/central-management/release-notes.md (hunk 6 above)
Comment (vexx32) [reply input/en-us/central-management/release-notes.md:32]: Resolving this for now so we can merge this and continue with the release.

**Comment by pauby** [comment]
@gep13 @vexx32 @AdmiringWorm There are a couple of issues with the release notes:

> - Add an option to provide a default value for the `executionTimoutInSeconds` setting, which will be used for all new Deployment Steps.

There is a spelling mistake in the value name.

> - Move the `Action` button to the first column in all tables.
> - Reinstate the ‘Return To Deployments’ button when editing a Deployment Plan.

This is inconsistent. Either buttons should be in a code block or shouldn't.

Note that @gep13 fixed both of these issues and updated the original notes in GitLab.

**Pull request merged**